src/components/urls/helpers.tsx

```diff
@@ -3,17 +3,17 @@ import { nonEmptyStr, nonEmptyArr } from '@subsocial/utils';
 import { BareProps } from '../utils/types';
 import copy from 'copy-to-clipboard';
 import { showInfoMessage } from '../utils/Message';
+import { appBaseUrl } from '../utils/env';
 
 export const openNewWindow = (url: string) => window.open(url, '_blank', 'toolbar=yes,scrollbars=yes,resizable=yes,top=500,left=500,width=400,height=400');
 
 export function slugify (text?: AnyText): string | undefined {
```

Comment:
Need to rename this func to make it looking more specific maybe: `slugifyHandle`?